src/utils.py

```diff
@@ -130,6 +126,41 @@ def get_battery_capacities(self):
 
         return capacities
 
+    # Register with landscape, returns True if successful
+    def register_landscape(self):
```

Comment:
Ultimately, it would be good to make both register_landscape and is_registered async so it doesn't block the UI. We can display a spinner while waiting for the processes to finish. But we can cover that in a later PR, this is a great incremental step and good enough for a first release.